### ISSUE TITLE
Deprecation warnings when installing RiderLink to UE-5.0.0-preview-2

### ISSUE THREAD

**Issue by slonopotamus** [JetBrains/UnrealLink#167]
```
[1m/tmp/JetBrainsPerUserTemp-1000-1/UnrealLink/Cogytij/HostProject/Plugins/RiderLink/Source/RD/src/rd_core_cpp/src/main/reactive/ViewableMap.h:31:35: [0m[0;1;35mwarning: [0m[1m'rebind<std::__1::pair<rd::Wrapper<int, std::__1::allocator<int>>, rd::Wrapper<int, std::__1::allocator<int>>>>' is deprecated [-Wdeprecated-declarations][0m
        using PA = typename VA::template rebind<std::pair<Wrapper<K>, Wrapper<V>>>::other;
[0;1;32m                                         ^
[0m[1m/ssd/unreal/ue-git/Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/type_traits:3157:38: [0m[0;1;30mnote: [0min instantiation of template class 'rd::ViewableMap<int, int, std::__1::allocator<int>, std::__1::allocator<int>>' requested here[0m
    : public integral_constant<bool, __is_constructible(_Tp, _Args...)>
[0;1;32m                                     ^
[0m[1m/ssd/unreal/ue-git/Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/type_traits:3225:14: [0m[0;1;30mnote: [0min instantiation of template class 'std::__1::is_constructible<rd::ViewableMap<int, int, std::__1::allocator<int>, std::__1::allocator<int>>, rd::ViewableMap<int, int, std::__1::allocator<int>, std::__1::allocator<int>> &&>' requested here[0m
    : public is_constructible<_Tp, typename add_rvalue_reference<_Tp>::type>
[0;1;32m             ^
[0m[1m/tmp/JetBrainsPerUserTemp-1000-1/UnrealLink/Cogytij/HostProject/Plugins/RiderLink/Source/RD/src/rd_core_cpp/src/main/reactive/ViewableMap.h:334:20: [0m[0;1;30mnote: [0min instantiation of template class 'std::__1::is_move_constructible<rd::ViewableMap<int, int, std::__1::allocator<int>, std::__1::allocator<int>>>' requested here[0m
static_assert(std::is_move_constructible<rd::ViewableMap<int, int>>::value, "Is move constructible from ViewableMap<int, int>");
[0;1;32m                   ^
[0m[1m/ssd/unreal/ue-git/Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/memory:1759:33: [0m[0;1;30mnote: [0m'rebind<std::__1::pair<rd::Wrapper<int, std::__1::allocator<int>>, rd::Wrapper<int, std::__1::allocator<int>>>>' has been explicitly marked deprecated here[0m
    template <class _Up> struct _LIBCPP_DEPRECATED_IN_CXX17 rebind {typedef allocator<_Up> other;};
[0;1;32m                                ^
[0m[1m/ssd/unreal/ue-git/Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/__config:1035:39: [0m[0;1;30mnote: [0mexpanded from macro '_LIBCPP_DEPRECATED_IN_CXX17'[0m
#  define _LIBCPP_DEPRECATED_IN_CXX17 _LIBCPP_DEPRECATED
[0;1;32m                                      ^
[0m[1m/ssd/unreal/ue-git/Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/__config:1012:48: [0m[0;1;30mnote: [0mexpanded from macro '_LIBCPP_DEPRECATED'[0m
#    define _LIBCPP_DEPRECATED __attribute__ ((deprecated))
[0;1;32m                                               ^
```

**Comment by slonopotamus** [comment]
Also:

```
Plugins/Developer/RiderLink/Source/RD/src/rd_core_cpp/src/main/reactive/ViewableSet.h:26:34: warning: 'rebind<rd::Wrapper<int, std::__1::allocator<int>>>' is deprecated [-Wdeprecated-declarations]
        using WA = typename A::template rebind<Wrapper<T>>::other;
                                        ^
Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/type_traits:3157:38: note: in instantiation of template class 'rd::ViewableSet<int, std::__1::allocator<int>>' requested here
    : public integral_constant<bool, __is_constructible(_Tp, _Args...)>
                                     ^
Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/type_traits:3225:14: note: in instantiation of template class 'std::__1::is_constructible<rd::ViewableSet<int, std::__1::allocator<int>>, rd::ViewableSet<int, std::__1::allocator<int>> &&>' requested here
    : public is_constructible<_Tp, typename add_rvalue_reference<_Tp>::type>
             ^
Plugins/Developer/RiderLink/Source/RD/src/rd_core_cpp/src/main/reactive/ViewableSet.h:270:20: note: in instantiation of template class 'std::__1::is_move_constructible<rd::ViewableSet<int, std::__1::allocator<int>>>' requested here
static_assert(std::is_move_constructible<rd::ViewableSet<int>>::value, "Is move constructible from ViewableSet<int>");
                   ^
Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/memory:1759:33: note: 'rebind<rd::Wrapper<int, std::__1::allocator<int>>>' has been explicitly marked deprecated here
    template <class _Up> struct _LIBCPP_DEPRECATED_IN_CXX17 rebind {typedef allocator<_Up> other;};
                                ^
Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/__config:1035:39: note: expanded from macro '_LIBCPP_DEPRECATED_IN_CXX17'
#  define _LIBCPP_DEPRECATED_IN_CXX17 _LIBCPP_DEPRECATED
                                      ^
Engine/Source/ThirdParty/Unix/LibCxx/include/c++/v1/__config:1012:48: note: expanded from macro '_LIBCPP_DEPRECATED'
#    define _LIBCPP_DEPRECATED __attribute__ ((deprecated))
                                               ^
```

**Comment by ArtemiyAAA** [comment]
Fixed in 2022.2/5.0.1 version
